packages/block-library/src/post-comments/style.scss

```diff
@@ -0,0 +1,33 @@
+
+.wp-block-post-comments {
+	.commentlist {
+		list-style: none;
+		margin: 0;
+		padding: 0;
+	}
+
+	.comment-author {
+		.avatar {
+			float: left;
+			margin-right: 10px;
```

Comment:
Should we use 1em here instead?

Reply:
The recent columns block has a `margin-right: .75em;` 🤔 https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/latest-comments/style.scss#L43
No matter what is decided, I think they should match.

Reply:
The avatar shape and size are also different 🤔